starboard/extension/media_codec_bridge.h

```diff
@@ -0,0 +1,45 @@
+// Copyright 2024 The Cobalt Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef STARBOARD_EXTENSION_MEDIA_CODEC_BRIDGE_H_
+#define STARBOARD_EXTENSION_MEDIA_CODEC_BRIDGE_H_
+
+#include <stdint.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define kStarboardExtensionMediaCodecBridgeName \
+  "dev.cobalt.extension.MediaCodecBridge"
+
+typedef struct StarboardExtensionMediaCodecBridgeApi {
```

Comment:
@y4vor to double check if this Starboard extension aligns to our general practice.

Reply:
Let's rename it to something more generic and Media related. Maybe `StarboardExtensionMediaSettingsApi`

Reply:
Renamed